src/components/Quiz/QuizWidget.tsx

```diff
@@ -283,141 +351,198 @@ const QuizWidget: React.FC<IProps> = ({ quizKey, maxQuestions }) => {
-                {showAnswer &&
-                  currentQuestionAnswerChoice &&
-                  !currentQuestionAnswerChoice.isCorrect && (
+                <Flex
+                  direction="column"
+                  alignItems="center"
+                  gap={{ base: 6, md: 2 }}
+                  mt={isStandaloneQuiz ? 8 : { md: 8 }}
+                >
+                  <Flex
+                    direction={{ base: "column", md: "row" }}
+                    gap={{ base: 4, md: 2 }}
+                    w="100%"
+                  >
                     <Button
-                      onClick={handleRetryQuestion}
                       variant="outline-color"
-                    >
-                      <Translation id="try-again" />
-                    </Button>
-                  )}
-                {showResults ? (
-                  <>
-                    <Button
                       leftIcon={<Icon as={FaTwitter} />}
                       onClick={handleShare}
                     >
                       <Translation id="share-results" />
                     </Button>
-                    {score < 100 && (
-                      <Button onClick={initialize}>
-                        <Translation id="try-again" />
+
+                    {/* Show `Next Quiz` button if quiz is opened from hub page */}
+                    {hasNextQuiz && (
+                      <Button onClick={handleNextQuiz}>
+                        <Translation id="next-quiz" />
                       </Button>
                     )}
-                  </>
-                ) : showAnswer ? (
-                  <Button onClick={handleContinue}>
-                    {userQuizProgress.length === quizData.questions.length - 1
-                      ? t("see-results")
-                      : t("next-question")}
-                  </Button>
-                ) : (
-                  <Button
-                    onClick={() =>
-                      handleShowAnswer(
-                        quizData.questions[currentQuestionIndex].id,
-                        currentQuestionAnswerChoice!
-                      )
-                    }
-                    isDisabled={!currentQuestionAnswerChoice}
-                  >
-                    <Translation id="submit-answer" />
-                  </Button>
-                )}
-              </Flex>
-            </Center>
-          </>
-        ) : (
-          <Flex justify="center">
-            <Spinner />
-          </Flex>
+                  </Flex>
+
+                  {showResults && (
+                    <Button
+                      onClick={initialize}
+                      variant="unstyled"
+                      color="primary"
+                      _hover={{ boxShadow: "none" }}
+                    >
+                      <Text textDecoration="underline" fontWeight="bold" m={0}>
+                        <Translation id="try-again" />
+                      </Text>
+                    </Button>
```

Comment:
Can we hide 'try again' button if they score full marks (100%)?

Looks straight forward, but if it causes any issues we can leave it out of this PR.

Reply:
yes, this was discussed on design sync and approved today

Reply:
Updated